knowledge/academy.rst

```diff
@@ -0,0 +1,10 @@
+.. _academy:
+
+Conan 2 Essentials Training
+===========================
+
+The Conan 2 Essentials training is available for free at the JFrog Academy.
```

Comment:
I would appreciate a bit more of detail, like for example that it is a hands-on training, with real exercises to learn by doing